### PR TITLE
Bump version to 17.0.1 & update changelog

### PR DESCRIPTION
**Pull request by rooftopcellist**
##### SUMMARY

There was a bug in the AWX Local Docker installer for 17.0.0 due to permissions not being right on the pgdocker directory.  This patch release is to fix that, and will also include a UI fix.  

Once the UI fix is in, I'll merge this and cut a release.  

##### ISSUE TYPE

 - Release Pull Request

##### AWX VERSION

```
17.0.1
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 This is a list of high-level changes for each release of AWX. A full list of commits can be found at `https://github.com/ansible/awx/releases/tag/<version>`.
 
+# 17.0.1 (January 26, 2021)
+- Fixed pgdocker directory permissions issue with Local Docker installer: https://github.com/ansible/awx/pull/9152
```
Comment (ryanpetrello) [inline CHANGELOG.md:6]: > and will also include a UI fix.

What's the UI fix?

Comment (rooftopcellist) [reply CHANGELOG.md:6]: @unlikelyzero do you have a link to the issue you mentioned?

Comment (unlikelyzero) [reply CHANGELOG.md:6]: https://github.com/ansible/awx/pull/9093

Comment (rooftopcellist) [reply CHANGELOG.md:6]: This just got merged, I updated the changelog and rebased.

**Review by jakemcdermott — APPROVED** [review]

**Pull request merged**